test/WebJobs.Script.Tests/Handlers/SystemTraceHandlerTests.cs

```diff
@@ -85,5 +87,15 @@ public void SetRequestId_SetsExpectedValue()
             requestId = request.GetRequestId();
             Guid.Parse(requestId);
         }
+
+        [Fact]
+        public void TraceWriter_ReturnsExpectedValue()
+        {
+            var instance = _handler.TraceWriter;
+            instance = _handler.TraceWriter;
+            instance = _handler.TraceWriter;
```

Comment:
Why are these lines duplicated?  If this is intentional, can you add a comment explaining what this test is doing?